src/assets/wise5/components/peerChat/peer-chat-question-bank/peer-chat-question-bank.component.ts

```diff
@@ -26,14 +26,20 @@ export class PeerChatQuestionBankComponent implements OnInit {
   @Output() displayedQuestionBankRulesChange = new EventEmitter<QuestionBankRule[]>();
   questions: string[];
 
-  constructor(private peerGroupService: PeerGroupService, private projectService: ProjectService) {}
+  constructor(
+    private constraintService: ConstraintService,
+    private peerGroupService: PeerGroupService,
+    private projectService: ProjectService
+  ) {}
 
   ngOnInit(): void {
     if (this.displayedQuestionBankRules == null) {
-      const referenceComponent = this.getReferenceComponent(this.content.questionBank);
+      const referenceComponent = this.projectService.getReferenceComponent(
+        this.content.questionBank
+      );
       if (
         this.content.questionBank.isPeerGroupingTagSpecified() &&
-        referenceComponent.content.type === 'OpenResponse'
+        ['OpenResponse', 'MultipleChoice'].includes(referenceComponent.content.type)
```

Comment:
Should the order of the component types be alphabetical like in the other locations? Ideally all these locations would just reference a single array somewhere that contains ['MultipleChoice', 'OpenResponse'].

Reply:
Hmm. Maybe in a ReferenceableComponents class or isReferenceableComponent() function?